h2/stream.py

```diff
@@ -1111,6 +1112,20 @@ def _lowercase_header_names(headers):
             yield (header[0].lower(), header[1])
 
 
+def _strip_surrounding_whitespace(headers):
+    """
+    Given an iterable of header two-tuples, strip both leading and trailing
+    whitespace from header both header names and header values. This generator
```

Comment:
Nitpicking: You have the word "header" here too many times. Consider changing to: "whitespace from both header names and header values."